src/PKSim.Core/Services/IApplicationSettingsPersitor.cs

```diff
@@ -2,5 +2,9 @@ namespace PKSim.Core.Services
 {
    public interface IApplicationSettingsPersitor : IPersistor<IApplicationSettings>
    {
```

Comment:
rename IApplicationSettingsPersitor ==> IApplicationSettingsPersi**s**tor

Reply:
yeah old stuff here. Nothing new with this PR